### ISSUE TITLE
Seppe Vansteelant: Use of ThirdParty

### ISSUE THREAD

**Issue by michaelgeamanu** [Informatievlaanderen/OSLOthema-consent#33]
Als ik het goed lees wordt DataProcessor als specialisatie van ThirdParty beschouwd? Dat is vreemd, een verwerker is net géén derde bij de verwerking. Derden zijn alle andere partijen dan verwerkingsverantwoordelijke, verwerker of betrokkene.

**Comment by GeertThijs** [comment]
We hebben hier bewust afgeweken van DPV om volgende redenen:

- Consent is iets tussen een Persoon en een DataController, je zou ze resp partij1 en partij2 kunnen noemen. Alle andere, waaronder de DataProcessor is semantisch gezien ThirdParty. 
- Uit GConsent gebruiken we Processing en om aan te geven wie de data verwerkt of anderzijds betrokken is bij de Processing is ere het attribuut Processing.involvesthirdParty.

Als DataProcessor echter wettelijk niet als een ThirdParty mag worden gezien moeten we dit aanpassen. Wel graag dan een verwijzing naar de overeenkomstige passage in de wetgeving.

**Comment by GeertThijs** [comment]
Ik denk dat dit correct is. In de remapping oefening (zie #46) hebben we dit aangepast: DataProcessor is geen subklasse meer van ThirdParty. Bij het remappen hebben we ThirdParty momenteel zelfs weggelaten, in de plaats hebben we nu wel een klasse Recipient, 1) geredeneerd dat dit in de meeste gevallen met ThirdParty wordt bedoeld en 2) om PersonalDataHandling compleet in overeenstemming met DPV uit te werken waar deze klasse ook voorkomt.

**Comment by GeertThijs** [comment]
Zie ook volgende classificatie in DPV:
![image](https://user-images.githubusercontent.com/22620291/192314629-c82da90a-ba7b-49fe-8541-2e93085d4cb9.png)

**Comment by SeppeVansteelant** [comment]
Inderdaad, in GDPR gaat het ook over 'ontvangers' of 'derde ontvangers'.
